lib/api/src/jsc/extern_ref.rs

```diff
@@ -0,0 +1,39 @@
+use std::any::Any;
```

Comment:
Can any of these files be made common between `jsc` and `js`? Perhaps using a `#[path()]` attribute?